components/radio/stories/template.js

```diff
@@ -15,6 +16,9 @@ export const Template = ({
 	isChecked = false,
 	isDisabled = false,
 	isReadOnly = false,
+	isFocused = false,
+	isInvalid = false,
```

Comment:
@aramos-adobe Based on that React link you sent me, I do think we need to pass the `isInvalid` arg to the radio components in field group. Right, they do not have the updated border colors. (I think I left a screenshot in my original note about this)

Thanks for sending that! 👍